### PR TITLE
Support the df command

### LINKED ISSUE #1642: generate /etc/mtab in container
Currently we do not construct /etc/mtab in the container so the df command does not function:
VIC:
```
root@ae7b8f307ee3:/# cat /etc/mtab
cat: /etc/mtab: No such file or directory
root@ae7b8f307ee3:/#
```

docker:
```
root@a1722c1e6629:/# cat /etc/mtab
none / aufs rw,relatime,si=f1cde4db423deea6,dio,dirperm1 0 0
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
tmpfs /dev tmpfs rw,nosuid,mode=755 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=666 0 0
sysfs /sys sysfs ro,nosuid,nodev,noexec,relatime 0 0
tmpfs /sys/fs/cgroup tmpfs ro,nosuid,nodev,noexec,relatime,mode=755 0 0
cgroup /sys/fs/cgroup/systemd cgroup ro,nosuid,nodev,noexec,relatime,xattr,release_agent=/lib/systemd/systemd-cgroups-agent,name=systemd,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/perf_event cgroup ro,nosuid,nodev,noexec,relatime,perf_event,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/net_cls,net_prio cgroup ro,nosuid,nodev,noexec,relatime,net_cls,net_prio,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/cpu,cpuacct cgroup ro,nosuid,nodev,noexec,relatime,cpu,cpuacct,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/memory cgroup ro,nosuid,nodev,noexec,relatime,memory,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/freezer cgroup ro,nosuid,nodev,noexec,relatime,freezer,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/hugetlb cgroup ro,nosuid,nodev,noexec,relatime,hugetlb,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/devices cgroup ro,nosuid,nodev,noexec,relatime,devices,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/pids cgroup ro,nosuid,nodev,noexec,relatime,pids,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/blkio cgroup ro,nosuid,nodev,noexec,relatime,blkio,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
cgroup /sys/fs/cgroup/cpuset cgroup ro,nosuid,nodev,noexec,relatime,cpuset,nsroot=/docker/a1722c1e66292e36453918865d643130db128c8e7f256b085e743d83f712e46b 0 0
mqueue /dev/mqueue mqueue rw,nosuid,nodev,noexec,relatime 0 0
/dev/mapper/ubuntu--1604--vmware--vg-root /var/lib/postgresql/data ext4 rw,relatime,errors=remount-ro,data=ordered 0 0
/dev/mapper/ubuntu--1604--vmware--vg-root /etc/resolv.conf ext4 rw,relatime,errors=remount-ro,data=ordered 0 0
/dev/mapper/ubuntu--1604--vmware--vg-root /etc/hostname ext4 rw,relatime,errors=remount-ro,data=ordered 0 0
/dev/mapper/ubuntu--1604--vmware--vg-root /etc/hosts ext4 rw,relatime,errors=remount-ro,data=ordered 0 0
shm /dev/shm tmpfs rw,nosuid,nodev,noexec,relatime,size=65536k 0 0
devpts /dev/console devpts rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000 0 0
proc /proc/bus proc ro,nosuid,nodev,noexec,relatime 0 0
proc /proc/fs proc ro,nosuid,nodev,noexec,relatime 0 0
proc /proc/irq proc ro,nosuid,nodev,noexec,relatime 0 0
proc /proc/sys proc ro,nosuid,nodev,noexec,relatime 0 0
proc /proc/sysrq-trigger proc ro,nosuid,nodev,noexec,relatime 0 0
tmpfs /proc/kcore tmpfs rw,nosuid,mode=755 0 0
tmpfs /proc/timer_stats tmpfs rw,nosuid,mode=755 0 0
tmpfs /proc/sched_debug tmpfs rw,nosuid,mode=755 0 0

```

### PR DESCRIPTION
**Pull request by anchal-agrawal**
* Symlinks `/proc/mounts` to `/etc/mtab` so `/bin/df` works
* Adds an integration test that is currently disabled due to #384 

Fixes #1642

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
ping @hickeng @caglar10ur @mhagen-vmware

File: tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.robot [hunk 1]
```diff
@@ -60,4 +60,11 @@ Docker run linked containers
     Run Keyword If  '${status}' == 'closed'  Fail  Test 1-6-Docker-Run.robot needs to be updated now that Issue #384 has been resolved
     Log  Issue \#384 is blocking implementation  WARN
     #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} run --link busy3:busy3 busybox ping -c2 busy3 
-    #Should Be Equal As Integers  ${rc}  0
\ No newline at end of file
+    #Should Be Equal As Integers  ${rc}  0
+
+Docker run df command
```
Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.robot]: ${status}=  Get State Of Github Issue  384

File: tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.md [hunk 2]
```diff
@@ -37,6 +38,7 @@ docker: Error response from daemon: Container command not found or does not exis
 docker: Error parsing reference: "fakeImage" is not a valid repository/tag.
 ```
 * Step 10 should result in success and the output should indicate that the ping succeeded across containers just using the linked name
+* Step 11 should result in success with exit code 0
```
Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.md]: and should successfully output the results of the df command

File: tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.robot (hunk 1 above)
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-6-Docker-Run.robot]: Whoops, sorry about that!

**Comment by mhagen-vmware** [comment]
lgtm

**Comment by caglar10ur** [comment]
Why did you close the original and opened a new one? LGTM

**Comment by anchal-agrawal** [comment]
@caglar10ur The previous PR was tagged [WIP] so it didn't trigger a CI build.

**Pull request merged**